index.js

```diff
@@ -184,7 +186,7 @@ AFRAME.registerComponent('teleport-controls', {
 
     if (!data.collisionEntities) {
       this.collisionEntities = [];
```

Comment:
You can set `.length = 0` instead

Reply:
Yep. Although I know it doesn't create a new array, it's also crazy slow compared with creating a new one, even if this code is not called plenty of times anyway: https://jsperf.com/initarraylength